packages/aws-cdk-lib/aws-ec2/lib/launch-template.ts

```diff
@@ -221,6 +221,15 @@ export interface LaunchTemplateProps {
    */
   readonly launchTemplateName?: string;
 
+  /**
+   * A description for the first version of the launch template.
+   * The version descrioption must be maximum 255 characters long, including hyphens (-),
+   * underscores (_), spaces, and tabs.
+   *
+   * @see http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-launchtemplate.html#cfn-ec2-launchtemplate-versiondescription
+   */
+  readonly versionDescription?: string;
```

Comment:
The CodeBuild CI seems to be failing. Optional properties must have `@default` documentation. Could you add the label and description?

```
aws-cdk-lib: error: [awslint:props-default-doc:aws-cdk-lib.aws_ec2.LaunchTemplateProps.versionDescription] Optional property must have @default documentation 
aws-cdk-lib: Error: /codebuild/output/src1195990305/src/github.com/aws/aws-cdk/tools/@aws-cdk/cdk-build-tools/bin/cdk-awslint exited with error code 1
aws-cdk-lib: Build failed.!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```